base/audit_events.go

```diff
@@ -61,7 +46,7 @@ var AuditEvents = events{
 		FilteringPermitted: false,
 		EventType:          eventTypeUser,
 	},
-	AuditIDUserAuthenticated: {
+	AuditIDPublicUserAuthenticated: {
 		Name:        "User authenticated",
```

Comment:
Do we want to do something to indicate that this is over the public port? in the name or description? See also below.

Reply:
Will tweak name and description in the follow-up full list of event definitions PR